application.py

```diff
@@ -1,9 +1,11 @@
+# import json
```

Comment:
leaving these comments to make it easier to test locally using this file from the command line